app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -174,11 +174,6 @@ function replaceActionButts(el) {
   wrapper.innerHTML = '<span class="current-user-actions">' + loggedInActionButts + '</span><a href="#" class="toggle-reply-form">REPLY</a>';
 }
 
-function warmNewCommentsArea() {
```

Comment:
This function is being called on when reply is being created and was returning 404 every other call. As per @benhalpern this can be removed.